src/libraries/System.Private.CoreLib/src/System/Threading/Tasks/ThreadPoolTaskScheduler.cs

```diff
@@ -45,17 +45,14 @@ protected internal override void QueueTask(Task task)
             if (Thread.IsThreadStartSupported && (options & TaskCreationOptions.LongRunning) != 0)
             {
                 // Run LongRunning tasks on their own dedicated thread.
-                Thread thread = new Thread(s_longRunningThreadWork);
-                thread.IsBackground = true; // Keep this thread from blocking process shutdown
-#if !TARGET_BROWSER
-                thread.Start(task);
-#endif
+#pragma warning disable CA1416 // TODO: https://github.com/dotnet/runtime/issues/44922
+                new Thread(s_longRunningThreadWork) { IsBackground = true }.UnsafeStart(task);
```

Comment:
I assume we can't name this ([as we do regular thread pool threads](https://github.com/dotnet/runtime/blob/master/src/libraries/System.Private.CoreLib/src/System/Threading/ThreadPool.cs#L1063)) as folks could have taken advantage of the fact that we start a new nameless thread and be doing the naming themselves?

Reply:
It would technically be a breaking change to add a name to a thread that didn't previously have one and on which user code runs.  Whether it would actually break apps in practice, I don't know.

Reply:
Yeah, I don't think it's worth it. It wasn't named in .NET Framework either.